### DISCUSSION TITLE
Neues Gerät hinzufügen: Speichern Button Schritt 4

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#28] [Shipped - Live on production]
Anstatt einen _Abschließen_ und einen _Speichern_ Button nur noch einen Button und den **Speichern & Abschließen** nennen

![Bildschirmfoto 2022-10-12 um 14 18 12](https://user-images.githubusercontent.com/80519295/195341254-11ad48a6-8cdc-4a20-ac88-d35d1de9367c.png)

**Comment by YvesR** [comment]
We do remove one button
